subprojects/snapshots/src/integTest/groovy/org/gradle/internal/vfs/VirtualFileSystemRetentionIntegrationTest.groovy

```diff
@@ -165,6 +191,7 @@ class VirtualFileSystemRetentionIntegrationTest extends AbstractIntegrationSpec
         executedAndNotSkipped ":compileJava", ":classes", ":run"
     }
 
+    // The `run` task is not yet supported for instant execution.
     @ToBeFixedForInstantExecution
```

Comment:
Should we add a `because` parameter to the annotation?

Reply:
Good idea, added.